test/qa/test_connect.py

```diff
@@ -468,4 +468,52 @@ def test_connect_to_unavailable_servers(tech, proto, obfuscated):
             sh.nordvpn.connect(name)
 
         print(ex.value)
-        assert lib.is_connect_unsuccessful(ex)
\ No newline at end of file
+        assert lib.is_connect_unsuccessful(ex)
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_status_disconnected(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    assert network.is_disconnected()
+    assert "Disconnected" in sh.nordvpn.status()
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(50)
+def test_status_connected(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    name, hostname = server.get_hostname_by(technology=tech, protocol=proto, obfuscated=obfuscated)
+    sh.nordvpn.connect(hostname.split(".")[0]) 
+    assert hostname in sh.nordvpn.status()
+
+    # Checking if correct IP address is shown in status
+    assert socket.gethostbyname(hostname) in sh.nordvpn.status()
+
+    assert "Connected" in sh.nordvpn.status()
+
+    assert server.get_country_name(name) in sh.nordvpn.status()
+    assert server.get_city_name(name) in sh.nordvpn.status()
+
+    assert tech.upper() in sh.nordvpn.status()
+
+    if tech == "openvpn":
+        assert proto.upper() in sh.nordvpn.status()
+    else:
+        assert "UDP" in sh.nordvpn.status()
+
+    transfer = str(sh.nordvpn.status()).split("\n")[7]
+    transfer_received = float(transfer.split(" ")[1])
+    transfer_sent = float(transfer.split(" ")[4])
+    assert transfer_received > 0
+    assert transfer_sent > 0
+
+    uptime = str(sh.nordvpn.status()).split("\n")[8]
+    time_connected = int(uptime.split(" ")[1])
+    assert time_connected > 0
```

Comment:
Same with this test, except it's easier to fix - can just count time since connect and check it with some leeway, like `assert time_connected > time_passed-1s && time_connected < time_passed+1s`